lib/screens/player_screen.dart

```diff
@@ -199,6 +214,97 @@ class _PlayerScreenContent extends StatelessWidget {
       ),
     );
   }
+
+  PageRouteBuilder _buildSlideRouteTransition(Widget sourceWidget, Widget targetWidget, {
+    RouteSettings? routeSettings,
+  }) {
+    return PageRouteBuilder(
+      settings: routeSettings,
+      pageBuilder: (context, animation, secondaryAnimation) => targetWidget,
+      transitionsBuilder: (context, animation, secondaryAnimation, child) {
+        const curve = Curves.ease;
+        const beginEnter = Offset(1.0, 0.0);
+        const endEnter = Offset.zero;
+        const beginExit = Offset(0.0, 0.0);
+        const endExit = Offset(-1.0, 0.0);
+
+        final tweenEnter = Tween(begin: beginEnter, end: endEnter);
+        final tweenExit = Tween(begin: beginExit, end: endExit);
+        final curvedAnimation = CurvedAnimation(
+          parent: animation,
+          curve: curve,
+        );
+
+        return Stack(
+          children: [
+            SlideTransition(
+              position: tweenExit.animate(curvedAnimation),
+              child: sourceWidget,
+            ),
+            SlideTransition(
+              position: tweenEnter.animate(curvedAnimation),
+              child: child,
+            ),
+          ],
+        );
+      },
+    );
+  }
+
+  Widget _buildBottomActions(BuildContext context, PlayerHideableController controller) {
+    return Consumer(
+      builder: (context, ref, child) {
+
+        final metadata = ref.watch(currentTrackMetadataProvider).unwrapPrevious();
+
+        final isLyricsLoading = metadata.isLoading || metadata.isRefreshing;
+        final isLyricsAvailable = (metadata.valueOrNull?.hasLyrics ?? false) && (metadata.valueOrNull?.lyrics != null || metadata.isLoading) && !metadata.hasError;
+        IconData getLyricsIcon() {
+          if (!isLyricsLoading && !isLyricsAvailable) {
+            return TablerIcons.microphone_2_off;
+          } else {
+            return TablerIcons.microphone_2;
+          }
+        }
+        
+        return Row(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            const Spacer(flex: 1,),
+            Expanded(
+              flex: 16,
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: [
+                  const SizedBox(
+                    width: 80,
+                    // child: Text("Output")
+                    child: SizedBox.shrink(),
+                  ),
+                  const SizedBox(
+                    width: 80,
+                    child: QueueButton()
+                  ),
+                  SizedBox(
```

Comment:
This widget is taller than the queue button due to the underlying TextButton not having its visualDensity set to -4 like the QueueButton.  This can break the layout and overflow.

Reply:
So I guess I just check the height difference between a regular button and one with visualDensity of -4, and then subtract that from the breakpoint heights?

Reply:
The breakpoint is tuned to the shorter queue button, so you need to increase it to match the lyric button.  Or shrink the lyric button instead.

Reply:
I've shrunken the button now. I don't like these implicit margins anyway. A visual density of -4 corresponds to a symmetric padding of 8px on each side by the way...